core/src/main/java/com/novoda/noplayer/exoplayer/forwarder/ExoPlayerForwarder.java

```diff
@@ -0,0 +1,74 @@
+package com.novoda.noplayer.exoplayer.forwarder;
+
+import com.novoda.noplayer.Player;
+import com.novoda.noplayer.PlayerState;
+import com.novoda.noplayer.listeners.BitrateChangedListeners;
+import com.novoda.noplayer.listeners.BufferStateListeners;
+import com.novoda.noplayer.listeners.CompletionListeners;
+import com.novoda.noplayer.listeners.ErrorListeners;
+import com.novoda.noplayer.listeners.InfoListeners;
+import com.novoda.noplayer.listeners.PreparedListeners;
+import com.novoda.noplayer.listeners.VideoSizeChangedListeners;
+
+public class ExoPlayerForwarder {
```

Comment:
Our generic forwarder that contains all of the `events` that we are interested in from `ExoPlayer`.